scripts/install-windows-prereqs.ps1

```diff
@@ -492,44 +464,39 @@ function Remove-DCAPDriver {
     return 0
 }
 
-
+# Starting from Intel SGX 2.11.101, the Intel SGX DCAP package contains both DCAP and PSW for Windows Server 2019 and Windows 10.
 function Install-DCAP-Dependencies {
-    Install-Tool -InstallerPath $PACKAGES["psw"]["local_file"] `
-                 -ArgumentList @('/auto', "$PACKAGES_DIRECTORY\Intel_SGX_PSW")
     Install-Tool -InstallerPath $PACKAGES["dcap"]["local_file"] `
                  -ArgumentList @('/auto', "$PACKAGES_DIRECTORY\Intel_SGX_DCAP")
 
-    if (($LaunchConfiguration -eq "SGX1FLC") -or ($DCAPClientType -eq "Azure"))
-    {
-        $drivers = @{
-            'sgx_base' = @{
-                'path'        = "$PACKAGES_DIRECTORY\Intel_SGX_PSW\Intel*SGX*PSW*\base\WindowsServer2019_Windows10"
-                'location'    = 'root\SgxLCDevice'
-                'description' = 'Intel(R) Software Guard Extensions Launch Configuration Service'
-            }
+    $drivers = @{
```

Comment:
why did you get rid of the if statetement at line 502?

Reply:
It seemed redundant as this function is called only if LaunchConfiguration is either SGX1FLC, or SGX1, or if DCAPClientType is Azure. Then in the function it does another check for LaunchConfiguration and DCAPClientType. The only case where the 2nd if statement at line 502 catches where the 1st doesn't is if LaunchConfiguration is SGX1 and DCAPClientType is None -- however in this case we still should be installing PSW and DCAP.

This where the function is called:
```
    if (($LaunchConfiguration -ne "SGX1FLC-NoIntelDrivers") -and ($LaunchConfiguration -ne "SGX1-NoIntelDrivers") -or ($DCAPClientType -eq "Azure")) {
        Install-DCAP-Dependencies
    }
```